Author: Stephan Peijnik
Comment:
  Added more usable error message when exiting due to python-wxgtk2.8 not being
  installed.

--- ipython-0.9.1.orig/IPython/frontend/wx/ipythonx.py	2008-10-29 11:50:42.777153341 +0100
+++ ipython-0.9.1/IPython/frontend/wx/ipythonx.py	2008-10-29 11:52:10.136154913 +0100
@@ -9,6 +9,9 @@
     e.message = """%s
 ________________________________________________________________________________
 You need wxPython to run this application.
+
+wxPython can be installed from the wxPython package. This can be
+done by running 'pisi install wxPython'.
 """ % e.message
     e.args = (e.message, ) + e.args[1:]
     raise e
